src/extractor/extractor_cmd.cpp

```diff
@@ -206,6 +206,10 @@ Command extractor_config_cmd() {
                     .help("sort frontier");
                 parser.add_argument<bool>("--neighbors-sorted")
                     .help("sort neighbors");
+                parser.add_argument<bool>("--dynamic-extraction")
```

Comment:
It is better that we sort rewrite these flags' help messages so that they become more understandable.